conda-forge.yml

```
@@ -3,5 +3,6 @@ appveyor:
 compiler_stack: comp7
 max_py_ver: '37'
 max_r_ver: '35'
+provider: {linux_aarch64: default, linux_ppc64le: default}
```

Should we try it with `native`?
https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/configure_feedstock.py#L1329
```suggestion
provider: {linux_aarch64: native, linux_ppc64le: default}
```

---

Wasn't aware of that!

---

I think the time limit is real tight. But this was some cool work done by @jmreicha and isuruf